pkg/config/defaults.go

```diff
@@ -47,6 +47,10 @@ func setDefaults(local bool) {
 	viper.SetDefault("auth.ui_config.login_failed_message", "The credentials don't match.")
 	viper.SetDefault("auth.ui_config.login_cookie_names", "internal_auth_session")
 
+	viper.SetDefault("auth.remote_authenticator.enabled", false)
```

Comment:
No need to set default to the flag's default

Reply:
💯